samples/pli/docker-compose.yml

```diff
@@ -0,0 +1,77 @@
+# example docker-compose.yml file for a Pli application
+
+# environment variables are defined in the adjacent .env file,
+# which may be used for allow developer-specific overrides
+
+# available environment variables:
+# - SOLUTION_ROOT (default=.) - relative/absolute path to the solution
+
+version: "3.7"
+
+# this file defines two services: web and db
+# services will typically be run together via `docker-compose up`, but
```

Comment:
While I understand this benefit for Windows devs, I don't understand how this is better for you and other non-Windows developers.  Do you run `docker-compose up` from within your windows VM, or do you use `docker-machine` to connect to a Windows host?

Did you consider putting the SQL server in Linux and the web server in a separate Windows Docker?

Reply:
I set `DOCKER_HOST` to point to my Windows Server VM. I don't actually have Docker for Mac installed, just the CLI tools. I did consider putting SQL Server in Linux, but it's actually a worse option for me with the current limitations of Docker for Windows:
- Currently unable to run Windows Containers and LCOW simultaneously, so HyperV would be needed (which means nested virtualization, so more memory usage)
- Could split the containers and run one on the Windows VM and one on the Mac host, but this has two downsides in itself:
  - Can't use `docker-compose`
  - Docker for Mac has to use a virtualization layer to run Linux VMs, so I would essentially be running two separate virtual machines, each running a single container, which seems less efficient than a single VM with two native containers

Reply:
I hope that you have the time to create a skeleton README at some point.

Reply:
I will do that before merging this PR - was just waiting for feedback before spending time documenting an approach that was disliked (so thank you!)